.github/workflows/cd.yml

```diff
@@ -122,11 +122,11 @@ jobs:
           then
               git push -d origin $(git tag --points-at HEAD | grep templates | grep rc)
           fi
-      
+
       - name: generate templates
         run: |
           curl -sSL https://git.io/get-mo -o mo && chmod +x mo && sudo mv mo /usr/local/bin/
-          .github/scripts/template-zip-autogen.sh
+          .github/scripts/template-zip-autogen.sh ${{ runner.temp }}/teamsfx_templates
```

Comment:
where locate the ${{runner.temp}} ?

Reply:
/home/runner/work/_temp
See https://docs.github.com/en/actions/learn-github-actions/contexts#runner-context